### DISCUSSION TITLE
Boss直聘推荐列表页，价格显示为方框

### DISCUSSION THREAD

**Discussion by wym1ng** [F9y4ng/GreasyFork-Scripts#379] [问答专区(Question / Answer)]
https://www.zhipin.com/web/geek/job-recommend
在"需要渲染的网页元素",添加 :not([class*='job-salary'])，数字不渲染可正常显示

**Comment by F9y4ng** [comment]
没账户无法查看。如有需要，请提供测试账户。

**Reply by wym1ng** [reply]
大佬，这是新的页面，两个页面都无需登录账号，看能否预设到预定义的渲染规则
https://www.zhipin.com/web/geek/jobs

**Reply by F9y4ng** [reply]
在**需要渲染的网页元素**中，追加`:not(.job-salary)`，保存为站点独享数据。
